testing/tests/DevExpress.ui.widgets.scheduler/appointments.tests.js

```diff
@@ -543,7 +543,7 @@ QUnit.test("Allday appointment should stay in allDayContainer after small draggi
     var $appointment = $("#allDayContainer .dx-scheduler-appointment"),
         pointer = pointerMock($appointment).start();
 
-    pointer.dragStart().drag(0, -30);
+    pointer.dragStart().drag(0, 30);
```

Comment:
Drop target is anchored under top-left corner. Dragging the appointment up would move the appointment to another row, so I had to change the dragging direction.